### ISSUE TITLE
question on parallel tree search

### ISSUE THREAD

**Issue by lcwd2** [jonathan-laurent/AlphaZero.jl#141]
AGZ uses virtual loss to make the tree search threads explore different paths. What is the strategy used in AlphaZero.jl?

**Comment by jonathan-laurent** [comment]
AlphaZero.jl does not have multiple workers explore the same MCTS tree and therefore there is no need for a virtual loss. Instead, many games are simulated in parallel during self-play and network inference requests are batched across those.

**Comment by lcwd2** [comment]
We probably still want a mechanism for matches to facilitate a large number of simulations.

**Comment by jonathan-laurent** [comment]
This is true and as a matter of fact, matches would be quite slow with the current implementation. This is on my TODO list though.

**Comment by lcwd2** [comment]
looking forward to the new release
